### PR TITLE
🔧 Create missing folders

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
RoR will now create any missing folders automatically

### CODE REVIEW

File: source/main/utils/PlatformUtils.h [hunk 1]
```diff
@@ -40,13 +40,15 @@ namespace RoR {
 
 extern char PATH_SLASH;
 
-bool FileExists(const char* path);   //!< Path must be UTF-8 encoded.
+bool FileExists(const char *path);   //!< Path must be UTF-8 encoded.
```
Comment (ulteq) [inline source/main/utils/PlatformUtils.h]: Clang-Format

**Comment by tritonas00** [comment]
Nice! I tried with no .rigsofrods dir to see what happens. It seems in that case, cache is not generated at all:

ca

**Comment by ulteq** [comment]
> I tried with no .rigsofrods dir

No `.rigsofrods` dir mean no content, right?

**Comment by ulteq** [comment]
Right!, @AnotherFoxGuy

**Comment by ulteq** [comment]
Let's merge this and fix the content issue with another pull request.

**Comment by AnotherFoxGuy** [comment]
> cache is not generated at all

That is an issue with missing .cfg files, but that is probably for another PR
Since some of the .cfg files are meant to be read-only so I don't see the point in having them
https://github.com/RigsOfRods/rigs-of-rods/blob/master/resources/skeleton/config/categories.cfg

**Comment by ulteq** [comment]
The problem is that we forgot to automatically unpack the `skeleton.zip` if needed.

**Pull request closed without merging**